simulator-web/src/utils/storage.ts

```diff
@@ -0,0 +1,21 @@
+import { useLocalStorage } from "react-use"
+
+const bufferToString = (buf: Uint8Array) => {
+  return String.fromCharCode(...new Uint8Array(buf))
+}
+
+const stringToUint8Array = (str: string) => {
+  const buf = new ArrayBuffer(str.length)
+  const bufView = new Uint8Array(buf)
+  for (let i = 0, strLen = str.length; i < strLen; i++) {
+    bufView[i] = str.charCodeAt(i)
+  }
+  return bufView
+}
+
+export const useLocalStorageUint8Array = (key: string, initialValue: Uint8Array | undefined) =>
```

Comment:
storing the pub and priv keys in localstorage

Reply:
Nice. I think this is where we'll want the per-instance UUID too.